http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/service/AuthProviderSpec.scala

```diff
@@ -121,15 +123,15 @@ class AuthProviderSpec
       val proxy = proxyWithAuthProvider(spyProvider)
 
       // create
-      val cluster1 = leo.createCluster(userInfo, project, cluster1Name, testClusterRequest).futureValue
+      val cluster1 = leo.createCluster(userInfo, project, cluster1Name, testClusterRequest).unsafeToFuture.futureValue
 
       // get status
-      val clusterStatus = leo.getActiveClusterDetails(userInfo, project, cluster1Name).futureValue
+      val clusterStatus = leo.getActiveClusterDetails(userInfo, project, cluster1Name).unsafeToFuture.futureValue
 
       cluster1 shouldEqual clusterStatus
 
       // list
-      val listResponse = leo.listClusters(userInfo, Map()).futureValue
+      val listResponse = leo.listClusters(userInfo, Map()).unsafeToFuture.futureValue
```

Comment:
might as well just do `unsafeRunSync` in these places?

Reply:
I did `unsafeToFuture.futureValue` in all the tests. `.futureValue` is a ScalaTest thing which uses implicit `PatienceConfig`s, etc. Maybe we could make the tests use `IO` at some point but I didn't want to change them too much.

Reply:
ah make sense...I didn't think of `PatienceConfig`...